src/eei.cpp

```diff
@@ -261,6 +261,8 @@ namespace hera {
 
       HERA_DEBUG << "getBlockHash " << hex << number << " " << resultOffset << dec << "\n";
 
+      ensureCondition(number >= 0 && number <= 256, ArgumentOutOfRange, "getBlockHash only accepts a block number between 0 and 256");
```

Comment:
That's not correct. It accepts range of `(current_block_number - 1, current_block_number - 257)` .

Reply:
Btw, we could also change `getblockhash` in EVMC to return an error code for reason of failure.

Reply:
https://github.com/ethereum/evmc/issues/63